Elevel/Application/Features/ApplicationUserFeatures/GetApplicationUserByIdQuery.cs

```diff
@@ -0,0 +1,71 @@
+﻿using AutoMapper;
+using Elevel.Application.Infrastructure;
+using Elevel.Application.Pagination;
+using Elevel.Domain.Models;
+using MediatR;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Elevel.Application.Features.ApplicationUserFeatures
+{
+    public class GetApplicationUserByIdQuery
+    {
+        public class Request : IRequest<Response>
+        {
+            public Guid Id { get; set; }
+        }
+
+        [Serializable]
+        public class Response
```

Comment:
please use same order for all files
1 request
2 handler 
3 response (+ additional response classes)